src/collective/cover/content.py

```diff
@@ -148,6 +149,26 @@ def get_referenced_objects(self):
                 refs |= getObjectsFromLinks(self, links)
         return refs
 
+    def purge_deleted_tiles(self):
+        """Check if there are any annotation of deleted tiles and destroy it.
+
+           Cover tiles are Browser Views, and data are saved in annotations.
+           When delete a tile, the annotations are not deleted, it keeps making
+           Cover object bigger;
+           This method purge all tile annotation data for tiles where there are
+           no reference in layout.
+        """
+        prefixes = ('plone.tiles.data', 'plone.tiles.configuration', 'plone.tiles.permission')
```

Comment:
these prefixes are defined as constants we should import and use the constants instead of hard-coding them.